examples/sycl/tbb-async-sycl/tbb-async-sycl.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
@omalyshe, should we add `sln`, `vcxproj` and `vcxproj.filters` files to the repo? As far as I remember, we CMake to generate .sln during testing.